eslint.config.js

```diff
@@ -70,6 +70,9 @@ function createInternalFilesPatterns(pattern = null) {
         }));
 }
 
+/**
+ * @type {import("eslint").Linter.Config[]}
```

Comment:
Should import from `lib/types` instead?

Reply:
It resolves to `lib/types/index` so they're the same thing but I'll change it to be more explicit.